style.css

```diff
@@ -0,0 +1,107 @@
+body {
+  font-family: Roboto, Helvetica, Arial, Verdana, Tahoma, sans-serif;
+}
+
+.wrapper {
+  width: 80%;
+  margin: auto;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+}
+
+.header__logo {
+  padding: 40px 0;
+  font-family: PT Serif, Helvetica, Arial, Verdana, Tahoma, sans-serif;;
```

Comment:
Codacy found an issue: [Expected newline after ";" in a multi-line declaration block (declaration-block-semicolon-newline-after)](https://app.codacy.com/gh/WebHeroSchool/LessonsHTML/pullRequest?prid=6226470)